vsintegration/tests/UnitTests/AssemblyResolver.fs

```diff
@@ -47,8 +46,6 @@ module AssemblyResolver =
             | None -> Unchecked.defaultof<Assembly>
             | Some name -> Assembly.Load(name) )
 
-[<SetUpFixture>]
 type public AssemblyResolverTestFixture () =
 
-    [<OneTimeSetUp>]
```

Comment:
From which files is this called now in a guaranteed way?

Reply:
Removed, not needed, see the other comment.